<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddResourceDialog</class>
 <widget class="QDialog" name="AddResourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Resource</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QLineEdit" name="pathEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="chooseFile_Button">
     <property name="text">
      <string>Browse...</string>
     </property>
     <property name="icon">
      <iconset resource="../MIDASDesktopIcons.qrc">
       <normaloff>:/icons/folder_add.png</normaloff>:/icons/folder_add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="localParentRadioButton">
     <property name="text">
      <string>Local parent</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="serverParentRadioButton">
     <property name="text">
      <string>Server parent ID:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="resourceTypeComboBox">
     <item>
      <property name="text">
       <string>Community</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Collection</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Item</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bitstream</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Resource Type:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLineEdit" name="parentIdEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>localParentRadioButton</tabstop>
  <tabstop>serverParentRadioButton</tabstop>
  <tabstop>parentIdEdit</tabstop>
  <tabstop>resourceTypeComboBox</tabstop>
  <tabstop>pathEdit</tabstop>
  <tabstop>chooseFile_Button</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../MIDASDesktopIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddResourceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddResourceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
